xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-server/src/main/java/org/xwiki/rest/internal/url/resources/JobStatusRestURLGenerator.java

```diff
@@ -46,7 +46,7 @@ public URL getURL(List<String> id) throws XWikiRestException
         // The idea is to use the UriBuilder of jax-rs to generate URLs that match the resources paths.
         // So it is consistent.
         try {
-            return Utils.createURI(getBaseURI(), JobStatusResource.class, getIdStringElement(id)).toURL();
+            return Utils.createURI(getBaseURI(), JobStatusResource.class, id).toURL();
```

Comment:
Seems to me the real fix is to pass `id.toArray()` here instead of a the `id` List, that way instead of passing a list as first element you pass several elements.